app/modules/users/models.py

```diff
@@ -618,3 +618,10 @@ def get_public_user(cls):
             full_name='Public User',
             is_internal=True,
         )
+
+    def get_sightings(self):
+        sightings = []
+        for encounter in self.owned_encounters:
+            if encounter.get_sighting() not in sightings:
```

Comment:
this is more me trying to learn python than anything, but could this instead be a **set** used here and then you not have to check for duplicates?    i used this once in some sightings code and hoping that i am correct in the assumption that `some_set.add(thing)` just silently does nothing if `thing` is already in the set.

Reply:
Do you mean a python dictionary? In which case, I think you're right.

Reply:
This seems to work fine:
`    def get_sightings(self):
        sightings = []
        for encounter in self.owned_encounters:
                sightings.append(encounter.get_sighting())
        sighting_set = set(sightings)
        return list(sighting_set)
`
Not sure if it wins us much, but it's already done so I'll leave it in.